city_scrapers/spiders/il_sports_facilities_authority.py

```diff
@@ -0,0 +1,105 @@
+from city_scrapers_core.constants import BOARD, COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class IlSportsFacilitiesAuthoritySpider(CityScrapersSpider):
+    name = "il_sports_facilities_authority"
+    agency = "Illinois Sports Facilities Authority"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.isfauthority.com/governance/board-meetings/"]
+    location = {
+        "name": "Authority offices",
+        "address": "Guaranteed Rate Field, 333 West 35th Street, Chicago, IL",
+    }
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        self._validate_location(response)
+
+        item = response.css(".wpb_wrapper .inner-text h2::text").getall()[0]
+        meeting = Meeting(
+            title=self._parse_next_title(item),
+            description="",
+            classification=BOARD,
+            start=self._parse_next_start(item),
+            end=None,
+            all_day=False,
+            time_notes="",
+            location=self.location,
+            links=[],
+            source=response.url,
+        )
+
+        meeting["status"] = self._get_status(meeting)
+        meeting["id"] = self._get_id(meeting)
+
+        yield meeting
+
+        for item in response.css(".wpb_wrapper p")[2:]:
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description="",
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes="",
+                location=self.location,
+                links=self._parse_links(item),
+                source=response.url,
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _validate_location(self, response):
+        location_text = response.css(".wpb_wrapper p strong::text").getall()[1]
+        if "Guaranteed Rate Field" not in location_text:
+            raise ValueError("Meeting location has changed")
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        parts = item.css("::text").get().split()
```

Comment:
Small thing, but for this and `_parse_next_title` we're trying to remove "Meeting" from the titles unless it's "Special Meeting" because it can be redundant in context. Meetings of the Board of Directors can be "Board of Directors" instead of just "Board" though

Reply:
Combined `_parse_next_title` and `_parse_title` in one function with commit 843ec5c.

For the current page, these are the meeting titles with their frequency:
```
     29 "Board of Directors"
      3 "Construction Committee"
      3 "Audit, Finance & Investment Committee"
      2 "Audit, Finance and Investment Committee"
      2 "Audit Finance and Investment Committee"
      1 "Executive Committee"
```

Reply:
Realized that "Special Board Meeting" was not treated correctly, fixed to look like follows for the current meetings:
```
     28 "Board of Directors"
      3 "Construction Committee"
      3 "Audit, Finance & Investment Committee"
      2 "Audit, Finance and Investment Committee"
      2 "Audit Finance and Investment Committee"
      1 "Special Board Meeting"
      1 "Executive Committee"
```